language/move-prover/src/cli.rs

```diff
@@ -344,6 +351,12 @@ impl Options {
                     .help("run the ABI generator instead of the prover. \
                     Generated ABIs will be written into the directory `./abi` unless configured otherwise via toml"),
             )
+            .arg(
+                Arg::with_name("errmapgen")
+                    .long("errmapgen")
```

Comment:
One more like this and I guess we need to make a real cli out of this, like `mc errmapgen <options`...